### PR TITLE
Post review

### PR DESCRIPTION
**Pull request by maxlin94**
Me and Oskar programmed together using live share.

### CODE REVIEW

**Review by oskber — APPROVED** [review]
Great, I will push some minor changes later.

**Comment by ConnyPettersson** [comment]
Looks nice!

**Review by ConnyPettersson — APPROVED** [review]
Looks nice!

**Review by MadeleineDahlqvist — APPROVED** [review]
Looks good to me, great work!

File: utils/reviewsUtils.js [hunk 1]
```diff
@@ -50,5 +50,13 @@ export function filterVerified(arr){
 
 //skriv här
 
-
-
+export async function postReview(review) {
+    const res = await fetch(`${API_BASE}/reviews`, {
+      method: 'POST',
```
Comment (richardolsson) [inline utils/reviewsUtils.js:55]: Indentation here is a bit sloppy. The first line has no indentation, the second has 4 spaces, and the third has 6 spaces. This means that the levels are not consistent. I would expect 0+2+4 or 0+4+8.

**Review by richardolsson — COMMENTED** [review]
This is mostly very good. I left one minor comment below.

**Pull request merged**